src/models/StyleSheet.js

```diff
@@ -3,6 +3,28 @@
 /* Wraps glamor's stylesheet and exports a singleton for the rest
 *  of the app to use. */
 
-import { StyleSheet } from '../vendor/glamor/sheet'
+import { StyleSheet as GlamorStyle } from '../vendor/glamor/sheet'
 
-export default new StyleSheet({ speedy: false, maxLength: 40 })
+class StyleSheet {
+  styleSheet: any
```

Comment:
@satya164 the line that @relekang commented is this, 
`import { StyleSheet as GlamorStyle } from '../vendor/glamor/sheet'`

from your suggestion it should be like this?
`import { StyleSheet as GlamorStyle: StyleSheet } from '../vendor/glamor/sheet'`  I'm getting token errors 😱